cmd/uffd-spike/main.go

```
@@ -0,0 +1,361 @@
+		}
+	}
+	_ = fc.Process.Kill()
+	_, _ = fc.Process.Wait()
+	return pre, nil
+}
+
+// prepareHeartbeatRootfs copies the base image and injects the heartbeat init
+// script via a loop mount (root only).
+func prepareHeartbeatRootfs(log *slog.Logger, base, scratch string) error {
+	log.Info("copying base rootfs", slog.String("from", base), slog.String("to", scratch))
+	if out, err := exec.Command("cp", "--reflink=auto", base, scratch).CombinedOutput(); err != nil {
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)